components/arm/client.go

```diff
@@ -112,18 +117,45 @@ func (c *client) Stop(ctx context.Context, extra map[string]interface{}) error {
 }
 
 func (c *client) ModelFrame() referenceframe.Model {
-	// TODO(erh): this feels wrong
-	return nil
+	return c.model
 }
 
 func (c *client) CurrentInputs(ctx context.Context) ([]referenceframe.Input, error) {
-	return nil, errArmClientInputsNotSupport
+	resp, err := c.JointPositions(ctx, nil)
+	if err != nil {
+		return nil, err
+	}
+	if c.model == nil {
+		return nil, errArmClientModelNotValid
+	}
+	return c.model.InputFromProtobuf(resp), nil
 }
 
 func (c *client) GoToInputs(ctx context.Context, goal []referenceframe.Input) error {
-	return errArmClientInputsNotSupport
+	if c.model == nil {
+		return errArmClientModelNotValid
+	}
+	return c.MoveToJointPositions(ctx, c.model.ProtobufFromInput(goal), nil)
 }
 
 func (c *client) DoCommand(ctx context.Context, cmd map[string]interface{}) (map[string]interface{}, error) {
 	return generic.DoFromConnection(ctx, c.conn, c.name, cmd)
 }
+
+func getModel(ctx context.Context, r robotpb.RobotServiceClient, name string) referenceframe.Model {
+	resp, err := r.FrameSystemConfig(ctx, &robotpb.FrameSystemConfigRequest{})
+	if err != nil {
+		return nil
+	}
+	cfgs := resp.GetFrameSystemConfigs()
+	for _, cfg := range cfgs {
+		part, err := config.ProtobufToFrameSystemPart(cfg)
+		if err != nil {
+			return nil
+		}
+		if part.Name == name {
```

Comment:
I believe this actually needs to be `.ShortName()`

Could you please ensure this is tested on an actual remote robot arm, in addition to the unit test?

Reply:
Yeah I was going to do a test later this afternoon to check this

Reply:
I tested this, turns out that this is not necessary